### PR TITLE
Fix `test_create_global_spn`

### LINKED ISSUE #1543: Test failure: `test_create_global_spn`
<details>
<summary>❌ test_create_global_spn: OSError: cannot get access token: ERROR: Please run 'az login' to setup account. (1m9.516s)</summary>

```
OSError: cannot get access token: ERROR: Please run 'az login' to setup account.
23:55 INFO [databricks.labs.ucx.mixins.fixtures] Schema hive_metastore.ucx_szwca: https://DATABRICKS_HOST/explore/data/hive_metastore/ucx_szwca
23:55 DEBUG [databricks.labs.ucx.mixins.fixtures] added schema fixture: SchemaInfo(browse_only=None, catalog_name='hive_metastore', catalog_type=None, comment=None, created_at=None, created_by=None, effective_predictive_optimization_flag=None, enable_predictive_optimization=None, full_name='hive_metastore.ucx_szwca', metastore_id=None, name='ucx_szwca', owner=None, properties=None, schema_id=None, storage_location=None, storage_root=None, updated_at=None, updated_by=None)
[gw2] linux -- Python 3.10.14 /home/runner/work/ucx/ucx/.venv/bin/python
23:55 INFO [databricks.labs.ucx.mixins.fixtures] Schema hive_metastore.ucx_szwca: https://DATABRICKS_HOST/explore/data/hive_metastore/ucx_szwca
23:55 DEBUG [databricks.labs.ucx.mixins.fixtures] added schema fixture: SchemaInfo(browse_only=None, catalog_name='hive_metastore', catalog_type=None, comment=None, created_at=None, created_by=None, effective_predictive_optimization_flag=None, enable_predictive_optimization=None, full_name='hive_metastore.ucx_szwca', metastore_id=None, name='ucx_szwca', owner=None, properties=None, schema_id=None, storage_location=None, storage_root=None, updated_at=None, updated_by=None)
[gw2] linux -- Python 3.10.14 /home/runner/work/ucx/ucx/.venv/bin/python
23:55 INFO [databricks.labs.ucx.mixins.fixtures] Schema hive_metastore.ucx_szwca: https://DATABRICKS_HOST/explore/data/hive_metastore/ucx_szwca
23:55 DEBUG [databricks.labs.ucx.mixins.fixtures] added schema fixture: SchemaInfo(browse_only=None, catalog_name='hive_metastore', catalog_type=None, comment=None, created_at=None, created_by=None, effective_predictive_optimization_flag=None, enable_predictive_optimization=None, full_name='hive_metastore.ucx_szwca', metastore_id=None, name='ucx_szwca', owner=None, properties=None, schema_id=None, storage_location=None, storage_root=None, updated_at=None, updated_by=None)
23:56 DEBUG [databricks.labs.ucx.mixins.fixtures] clearing 0 cluster policy fixtures
23:56 DEBUG [databricks.labs.ucx.mixins.fixtures] clearing 1 schema fixtures
23:56 DEBUG [databricks.labs.ucx.mixins.fixtures] removing schema fixture: SchemaInfo(browse_only=None, catalog_name='hive_metastore', catalog_type=None, comment=None, created_at=None, created_by=None, effective_predictive_optimization_flag=None, enable_predictive_optimization=None, full_name='hive_metastore.ucx_szwca', metastore_id=None, name='ucx_szwca', owner=None, properties=None, schema_id=None, storage_location=None, storage_root=None, updated_at=None, updated_by=None)
```

</details>

<sub>Running from [nightly #39](https://github.com/databrickslabs/ucx/actions/runs/8824812913/job/24227960176)</sub>

### PR DESCRIPTION
**Pull request by nkvuong**
## Changes
- This test should not run

### Linked issues

Resolves #1543

### Tests

- [ ] manually tested
- [ ] verified on staging environment (screenshot attached)

### CODE REVIEW

File: tests/integration/azure/test_access.py [hunk 1]
```diff
@@ -82,6 +82,7 @@ def test_create_global_spn(
     # skip this test if not in local mode
     if os.path.basename(sys.argv[0]) not in {"_jb_pytest_runner.py", "testlauncher.py"}:
         return
+    print(os.path.basename(sys.argv[0]))
```
Comment (nfx) [inline tests/integration/azure/test_access.py:85]: ?...

**Review by nfx — CHANGES_REQUESTED** [review]

File: tests/integration/azure/test_access.py (hunk 1 above)
Comment (nkvuong) [reply tests/integration/azure/test_access.py:85]: attempt to debug github actions 😭

Comment (nfx) [reply tests/integration/azure/test_access.py:85]: ah

File: tests/integration/azure/test_access.py [hunk 2]
```diff
@@ -65,6 +65,9 @@ def test_save_spn_permissions_local(ws, sql_backend, inventory_schema, make_rand
 
 @pytest.fixture
 def clean_up_spn():
+    # skip this fixture if not in local mode
+    if os.path.basename(sys.argv[0]) not in {"_jb_pytest_runner.py", "testlauncher.py"}:
+        return
```
Comment (nfx) [inline tests/integration/azure/test_access.py:70]:
```suggestion
def clean_up_spn(env_or_skip):
    # Making sure this test can only be launched from local
    env_or_skip("IDE_PROJECT_ROOTS")
```

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**